content/docs/querying/functions.md

```diff
@@ -184,6 +184,11 @@ rate(http_requests_total{job="api-server"}[5m])
 
 `rate` should only be used with counters.
 
+Note that when combining `rate()` with an aggregation operator (e.g. `sum()`)
+or a function aggregating over time (any function ending in `_over_time`),
+always take a `rate()` first, then aggregate, otherwise `rate()` cannot detect
```

Comment:
Run-on sentence. Either add an "as" after the "aggregate," or end the sentence there and start a new one.

:+1: otherwise.